integration/templates/astro-node/src/layouts/ViewTransitionsLayout.astro

```diff
@@ -17,7 +17,7 @@ const { title } = Astro.props;
     <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
     <meta name="generator" content={Astro.generator} />
     <title>{title}</title>
-    <ViewTransitions />
+    <ClientRouter />
```

Comment:
Previously `<ViewTransitions />` in Astro v4, now `<ClientRouter />` in v5. No action needed on our end.

https://docs.astro.build/en/guides/upgrade-to/v5/#renamed-viewtransitions--component